MVP/jenkins-jobs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+need_pip()
+{
+    echo "  We need 'pip' to install/update jenkins."
+    echo "  Please run 'easy_install pip' as root to install it."
+    exit 1
+}
+
+upgrade_jjb()
+{
+    which pip || need_pip
+    pip install --user --upgrade pip
+    pip install --user --upgrade jenkins-job-builder
```

Comment:
This script currently works fine on my side (1.4.0 atm) but I surely agree there is potential risk.
Haven't got a better idea than hardcoding it as you suggested. I'll start with it first.
Thanks!